packages/xstate-react/test/types.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { Machine, interpret, assign, spawn, Interpreter } from 'xstate';
+import { useService } from '../src';
+
+describe('useService', () => {
+  it('should accept spawned machine', () => {
```

Comment:
it's somewhat verbose, but I've wanted to replicate the whole situation - from spawning to using service 🤷‍♂